infrastructure/prod.tfvars

```diff
@@ -9,6 +9,7 @@ elastic_search_enabled = "true"
 
 data_store_max_pool_size = 48
 database_sku_name = "GP_Gen5_8"
+database_sku_capacity = "8"
```

Comment:
shouldn't AAT as well match prod?

Reply:
Yes, probably.  Let me check what it is now.

Reply:
Adjusted both demo and aat to match prod.  saat and sandbox will be default 2 core.